src/proto/generation.proto

```diff
@@ -24,6 +24,7 @@ enum ArtifactType {
   ARTIFACT_LATENT = 8;
   ARTIFACT_TENSOR = 9;
   ARTIFACT_DEPTH = 10;
+  ARTIFACT_MODEL = 11;  
```

Comment:
I don't think this is used, is it? May want to hold off on adding until have actual use in the API that needs it.

Reply:
That is correct, actually...we can take this out.